resources/app_dynamics_agent/defaults/conf/app-agent-config.xml

```diff
@@ -0,0 +1,702 @@
+<app-agent-configuration>
```

Comment:
This file isn't quite the equivalent of the contents of the download with one stanza added.  I'll make a polishing commit to bring it inline.

Reply:
Thanks @nebhale : Please make sure to get PR#699 as well.

The filter should be 

```
        <sensitive-data-filter applies-to="environment-variables,system-properties"
                               match-type="STARTSWITH"
                               match-pattern=""/>
```
not 

```
        <sensitive-data-filter applies-to="environment-variables,system-properties"
                               match-type="STARTSWITH"
                               match-pattern="*"/>
```